Bindings/Java/Matlab/Hopper_Device/BuildDevice.m

```diff
@@ -0,0 +1,162 @@
+%-----------------------------------------------------------------------%
+% The OpenSim API is a toolkit for musculoskeletal modeling and         %
+% simulation. See http://opensim.stanford.edu and the NOTICE file       %
+% for more information. OpenSim is developed at Stanford University     %
+% and supported by the US National Institutes of Health (U54 GM072970,  %
+% R24 HD065690) and by DARPA through the Warrior Web program.           %
+%                                                                       %
+% Copyright (c) 2017 Stanford University and the Authors                %
+% Author(s): Thomas Uchida, Chris Dembia, Carmichael Ong, Nick Bianco,  %
+%            Shrinidhi K. Lakshmikanth, Ajay Seth, James Dunne          %
+%                                                                       %
+% Licensed under the Apache License, Version 2.0 (the "License");       %
+% you may not use this file except in compliance with the License.      %
+% You may obtain a copy of the License at                               %
+% http://www.apache.org/licenses/LICENSE-2.0.                           %
+%                                                                       %
+% Unless required by applicable law or agreed to in writing, software   %
+% distributed under the License is distributed on an "AS IS" BASIS,     %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or       %
+% implied. See the License for the specific language governing          %
+% permissions and limitations under the License.                        %
+%-----------------------------------------------------------------------%
+function device = BuildDevice(varargin)
```

Comment:
I've added descriptions of parameters for all functions that we expect users to call. Some functions are not intended for direct use by users, so I did not add detailed MATLAB "help" docs for these (they simply say that the files are for use by the interactive GUI).

To get the help docs to show up properly, I had to move the license block below the help docs.